homeassistant/components/ombi/__init__.py

```diff
@@ -61,36 +92,58 @@ def setup(hass, config):
 
     def submit_movie_request(call):
         """Submit request for movie."""
-        name = call.data.get("name")
+        name = call.data.get(CONF_NAME)
```

Comment:
Please use `dict[key]` for required service data keys and keys with default schema values.